centaurCwlRunner/src/main/scala/centaur/cwl/OutputManipulator.scala

```diff
@@ -95,8 +99,17 @@ object OutputManipulator extends Poly1 {
     // We need this because some task return a "File" object that actually does not exist on disk but has its content provided directly instead
     def sizeContent: Option[Long] = obj.kleisli("contents").flatMap(_.asString.map(_.length.toLong))
 
-    // The cwl test runner expects only the name, not the full path
-    val updatedLocation = obj.add("location", Json.fromString(path.name))
+    val cleanName = callRootPaths.find(path.startsWith) match {
+      case Some(localRoot: DefaultPath) =>
+        // For local paths, the root doesn't have "execution" in it but that's where the outputs are
```

Comment:
🙈 
I'm just about to write the same thing to localize IWDR content to the execution directory 😢

Reply:
Oh this is actually not related to test 86 I didn't mean to push this here I must have mixed up my branches. This is for 106.
I might as well leave it here though..
And yeah this is far from great. Maybe we could have a `callExecutionRoot` on top of the `callRoot` published to metadata ?

Reply:
Yeah we might want to publish that. Currently I'm assuming in about 50 various places that the execution directory is immediately under the call directory but it's possible that could change. 😄